### PR TITLE
Upgrade Kubernetes CLI to 1.10.3 version

### PR DESCRIPTION
**Pull request by alnikyur**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -13,8 +13,8 @@
 
 
 IMAGE_NAME := instrumentisto/gitlab-builder
-VERSION ?= 0.4.2-docker18.05.0-compose1.21.2-kubectl1.10.2-helm2.9.1
-TAGS ?= 0.4.2-docker18.05.0-compose1.21.2-kubectl1.10.2-helm2.9.1,0.4.2,0.4,latest
```
Comment (tyranron) [inline Makefile:17]: @alnikyur why did you change Docker CLI version here if MR is related only to Kubernetes CLI?

**Review by tyranron — CHANGES_REQUESTED** [review]

File: Makefile (hunk 1 above)
Comment (alnikyur) [reply Makefile:17]: @tyranron See on your [commit](https://github.com/instrumentisto/gitlab-builder-docker-image/commit/7a4a4d17c9fead00d4c7ba00080838d5a9144383) and think about update, but now understood that is my mistake. I`ll fix it now.

**Review by tyranron — APPROVED** [review]

**Comment by tyranron** [comment]
@alnikyur now, please, apply the correct Git tag to `master` branch, push it, and create appropriate release on Github.

**Comment by alnikyur** [comment]
@tyranron `0.4.2-docker18.05.0-compose1.21.2-kubectl1.10.3-helm2.9.1` is this a correct tag name?

**Comment by tyranron** [comment]
@alnikyur yup, you just simply copy-paste version in `Makefile`.
